spark_ec2.py

```diff
@@ -76,10 +76,13 @@
     "1.5.1",
     "1.5.2",
     "1.6.0",
+    "1.6.1",
     "1.6.2",
+    "1.6.3,
```

Comment:
Pushed a fix in https://github.com/amplab/spark-ec2/commit/5188c78a0e9e5d34164ccbe1a361aeaac653bcc0